doc/versioning.rst

```diff
@@ -82,3 +82,14 @@ Example
   | oneAPI provisional 1.1 rev 2
   | oneAPI provisional 1.1 rev 3
   | oneAPI 1.1 rev 1
+
+Programmatic versioning
+=======================
+
+The specification version as mentioned above can be used programatically by appending all digits of the specification version in the format of <MAJOR><MINOR><REVISION>. MINOR and REVISION version use two digits. This versioning can be used to check the compatibility of the implementation with the specification version.
+
+Example
+  | oneAPI 1.1 rev 1 will be represented as a numerical value of 10101
+  | oneAPI 1.2 rev 1 will be represented as a numerical value of 10102
```

Comment:
According to the proposed format shouldn't  `1.2 rev 1` be `10201` here instead of `10102` since `2` is a minor?

Reply:
Fixed